config/Services/Storage.cs

```
@@ -216,6 +237,38 @@ public async Task<Package> GetPackageAsync(string id)
             return this.CreatePackageServiceModel(response);
         }
 
+        public async Task<PackageConfigurations> GetAllConfigurationsAsync()
+        {
+            try
+            {
+                var response = await this.client.GetAsync(PACKAGES_COLLECTION_ID, PACKAGES_CONFIGURATIONS_KEY);
+                return JsonConvert.DeserializeObject<PackageConfigurations>(response.Data);
+            }
+            catch (Exception)
+            {
+                //TODO: Logging
+            }
+            return new PackageConfigurations(); //Return empty response 
+            
+        }
+
+        public async Task UpdateConfigurationsAsync(string customConfigType)
+        {
+            PackageConfigurations list;
+            try
+            {
+                var response = await this.client.GetAsync(PACKAGES_COLLECTION_ID, PACKAGES_CONFIGURATIONS_KEY);
+                list = JsonConvert.DeserializeObject<PackageConfigurations>(response.Data);
+            }
+            catch (Exception) 
+            {
+                //TODO: logging
```

do you want to fail silently here? Does this mean you could lose configurations if you hit this error?

---

This means we currently do not have any configurations in DB. So in the call below, we will upsert the new configuration.